test/process_group_dist_autograd_test.py

```diff
@@ -0,0 +1,15 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import dist_autograd_test
+import process_group_rpc_agent_test_fixture
+
+
+class ProcessGroupDistAutogradTest(
```

Comment:
Why do we have test files specific to `ProcessGroup`? This would be hard to maintain as we add more RPC backends. Why can't we just have a single test file and enable/disable tests based on backends as we used to do before?

Reply:
Answered.